kytos/core/config.py

```diff
@@ -145,6 +145,10 @@ def parse_args(self):
 
         defaults.update(dict(config.items("daemon")))
 
+        # Gets vlan_pool after default is updated with values
+        # from config gile. then parses vlan_pool via json.loads
+        defaults.update(vlan_pool=json.loads(defaults.get('vlan_pool')))
```

Comment:
A final check that needs to be made is if the "vlan_pool" value is really being read when defined in `kytos.conf`.